bundles/org.openhab.binding.meteofrance/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -114,6 +141,9 @@
 		<label>Avalanches</label>
 		<description>Avalanche alert level</description>
 		<category>oh:meteofrance:avalanches</category>
+		<tags>
+			<tag>Alarm</tag>
```

Comment:
"Precipitation" missing

Reply:
Hmm. Generally the word 'precipitation' means stuff (which is not rain) falling from the sky (e.g. snow, hail, sleet..) .. so it is probably bending the rules somewhat to tag avalanches as 'precipitation'..

Reply:
Ok